web/client/components/map/leaflet/plugins/WMTSLayer.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2015, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const Layers = require('../../../../utils/leaflet/Layers');
+const CoordinatesUtils = require('../../../../utils/CoordinatesUtils');
+const WMSUtils = require('../../../../utils/leaflet/WMSUtils');
+const L = require('leaflet');
+const objectAssign = require('object-assign');
+const {isArray, isEqual} = require('lodash');
+const SecurityUtils = require('../../../../utils/SecurityUtils');
+
+
+L.TileLayer.WMTS = L.TileLayer.extend({

```

Comment:
No, just move the L.TileLayer.WMTS definition to an external file and require it from here